tests/integration/pilot/routing_test.go

```diff
@@ -19,14 +19,14 @@ import (
 	"testing"
 
 	"istio.io/istio/pkg/test/framework"
-	"istio.io/istio/tests/integration/pilot/common"
 )
 
 func TestTraffic(t *testing.T) {
 	framework.
 		NewTest(t).
 		Features("traffic.routing", "traffic.reachability", "traffic.shifting").
 		Run(func(ctx framework.TestContext) {
-			common.RunTrafficTest(ctx, apps)
+			ctx.FailNow()
+			//common.RunTrafficTest(ctx, apps)
```

Comment:
Just skip the test?

Reply:
The point of this PR is to dump debugging info. We don't store artifacts in CI unless we fail.